src/game/lara/lara_state.c

```diff
@@ -80,6 +81,8 @@ void Lara_State_Walk(ITEM_INFO *item, COLL_INFO *coll)
 
 void Lara_State_Run(ITEM_INFO *item, COLL_INFO *coll)
 {
+    static bool jump_permitted = true;
```

Comment:
I think this should be declared at the top of the like `static bool m_JumpPermitted = true;`.

Reply:
Thanks, do we prefer to avoid static variables in functions? I can't think that I've seen any others now that I think about it. I don't suppose scope really matters for this.

Reply:
Yes we want to make the global state easily seen this way.

Reply:
Got it, now moved to module scope and renamed accordingly.